scalabel/label/from_mmpkl.py

```diff
@@ -0,0 +1,645 @@
+"""Convert kitti to Scalabel format."""
+import argparse
+import copy
+import math
+import os
+import os.path as osp
+from typing import Dict, List, Tuple
+
+import numpy as np
+from PIL import Image
+from scipy.spatial.transform import Rotation as R
+
+from ..common.parallel import NPROC
+from ..common.typing import NDArrayF64
+from ..label.transforms import xyxy_to_box2d
+from ..label.utils import cart2hom, rotation_y_to_alpha
+from .io import save
+from .kitti_utlis import KittiPoseParser, list_from_file, read_calib, read_oxts
+from .typing import (
+    Box3D,
+    Category,
+    Config,
+    Dataset,
+    Extrinsics,
+    Frame,
+    FrameGroup,
+    ImageSize,
+    Intrinsics,
+    Label,
+)
+from .utils import (
+    get_box_transformation_matrix,
+    get_extrinsics_from_matrix,
+    get_matrix_from_extrinsics,
+)
+import numpy as np
+import struct
+from open3d import *
+import mmengine
+
+glyd_cats = {
+    "Glydcar": "car",
+    "Dummy": "dummy",
+}
+
+kitti_used_cats = ["Glydcar"]
+
+
+def parse_arguments() -> argparse.Namespace:
+    """Parse the arguments."""
+    parser = argparse.ArgumentParser(description="MMDetection to scalabel")
+    parser.add_argument(
```

Comment:
Seems this will be overrided by dir anyways?  Should we remove it here?

Reply:
Yes, removed